edgecast/client/auth_header_provider.go

```diff
@@ -5,7 +5,10 @@ package client
 import (
```

Comment:
This file can be deleted, its contents have been refactored as the `auth` package. That was my bad, an oversight while merging code.

Reply:
Thanks @stevenpaz ! Removed.